config/routes.rb

```diff
@@ -64,8 +65,19 @@
   # Api related routing
   namespace :api, :defaults => {:format => :json} do
     get 'games' => 'api#games'
-    get 'dashboard_data' => 'api#dashboard'
-    get 'dashboard_data/:game_id' => 'api#dashboard'
+    get 'dashboard_data' => 'api#dashboard', mobile: :mobile
```

Comment:
Here are the queries so we can pass for whether the dashboard is being viewed on desktop or mobile. Again, this currently does nothing.